desc/optimize/optimizer.py

```diff
@@ -296,9 +298,12 @@ def optimize(  # noqa: C901 - FIXME: simplify this
                         nonlinear_constraint.dim_f - num_equality
                     )
                 )
+        timer.stop("Initializing the optimization")
+        if verbose > 1:
+            timer.disp("Initializing the optimization")
 
         if verbose > 0:
-            print("Starting optimization")
+            print("\nStarting optimization")
```

Comment:
why do we need the extra newline here?

Reply:
We print many things, I felt like dividing the initialization and optimization part would be more organized. Using the divider like the `--------`s is a bit too much but I guess it makes it cleaner

Reply:
Don't you think this easier to focus?
```
Building objective: Bootstrap current self-consistency (Redl)
Precomputing transforms
Timer: Precomputing transforms = 832 ms
Timer: Objective build = 2.55 sec
Building objective: lcfs R
Building objective: lcfs Z
Building objective: fixed Psi
Building objective: fixed current
Building objective: fixed electron density
Building objective: fixed electron temperature
Building objective: fixed ion temperature
Building objective: fixed atomic number
Building objective: fixed sheet current
Building objective: self_consistency R
Building objective: self_consistency Z
Building objective: lambda gauge
Building objective: axis R self consistency
Building objective: axis Z self consistency
Timer: Objective build = 1.50 sec
Timer: Linear constraint projection build = 3.07 sec
Number of parameters: 37
Number of objectives: 2
Timer: Initializing the optimization = 7.39 sec

Starting optimization
Using method: lsq-exact
   Iteration     Total nfev        Cost      Cost reduction    Step norm     Optimality   
       0              1          7.871e-13                                    2.292e-13   
`gtol` condition satisfied.
         Current function value: 7.871e-13
         Total delta_x: 0.000e+00
         Iterations: 0
         Function evaluations: 1
         Jacobian evaluations: 1
Timer: Solution time = 9.66 sec
Timer: Avg time per step = 9.66 sec

--------------------------------------------------------------------------------------------------------------
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
--------------------------------------------------------------------------------------------------------------

                                                                 Start  -->   End
Total (sum of squares):                                      7.871e-13  -->   7.871e-13, 
Maximum absolute Bootstrap current self-consistency error:   1.013e-01  -->   1.013e-01 (T A m^-2)
Minimum absolute Bootstrap current self-consistency error:   1.010e-01  -->   1.010e-01 (T A m^-2)
Average absolute Bootstrap current self-consistency error:   1.012e-01  -->   1.012e-01 (T A m^-2)
Maximum absolute Bootstrap current self-consistency error:   1.256e-06  -->   1.256e-06 (normalized)
Minimum absolute Bootstrap current self-consistency error:   1.253e-06  -->   1.253e-06 (normalized)
Average absolute Bootstrap current self-consistency error:   1.255e-06  -->   1.255e-06 (normalized)
R boundary error:                                            0.000e+00  -->   0.000e+00 (m)
Z boundary error:                                            0.000e+00  -->   0.000e+00 (m)
Fixed Psi error:                                             0.000e+00  -->   0.000e+00 (Wb)
Fixed current profile error:                                 0.000e+00  -->   0.000e+00 (A)
Fixed electron density profile error:                        0.000e+00  -->   0.000e+00 (m^-3)
Fixed electron temperature profile error:                    0.000e+00  -->   0.000e+00 (eV)
Fixed ion temperature profile error:                         0.000e+00  -->   0.000e+00 (eV)
Fixed atomic number profile error:                           0.000e+00  -->   0.000e+00 (dimensionless)
Fixed sheet current error:                                   0.000e+00  -->   0.000e+00 (~)

--------------------------------------------------------------------------------------------------------------
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
--------------------------------------------------------------------------------------------------------------
```